charts/finrem-ns/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: finrem-ns
 apiVersion: v1
 home: https://github.com/hmcts/finrem-notification-service
-version: 0.0.6
+version: 0.0.7
```

Comment:
No need to bump Chart version unless you modify Chart infra files

Reply:
Reverted back to 0.0.6 👍